CMakeLists.txt

```diff
@@ -145,8 +145,11 @@ if(WIN32)
     add_definitions(-DBOOST_INTERPROCESS_BOOTSTAMP_IS_SESSION_MANAGER_BASED)
 endif()
 
-add_definitions(-DBOOST_CHRONO_HEADER_ONLY -DBOOST_CONFIG_SUPPRESS_OUTDATED_MESSAGE)
+add_definitions(-DBOOST_CHRONO_HEADER_ONLY -DBOOST_CONFIG_SUPPRESS_OUTDATED_MESSAGE -DBOOST_FILESYSTEM_NO_CXX20_ATOMIC_REF)
 
+if(APPLE)
+    add_definitions("-fno-aligned-allocation")
+endif()
```

Comment:
I think this should be applied more selectively - only on affected platforms (macOS <10.14) and probably only for affected code. 
This seems to work: https://github.com/dyfer/supercollider/commit/f458244f72302a0c045b7ee7ba171db60a56a121

Reply:
I think you are right. As you can see from the history in the "patched" version I was kind of poking in the dark since I have zero experience with MacOS based systems.